controllers/pwa.js

```diff
@@ -118,6 +114,7 @@ router.post('/add', (req, res, next) => {
       return pwaLib.createOrUpdatePwa(pwa);
     })
     .then(savedData => {
+      cache.flush();
```

Comment:
Does this flush the whole cache? Maybe we flush only this PWA and the recently added pages.